esphome/components/seeed_mr60bha2/binary_sensor.py

```diff
@@ -0,0 +1,25 @@
+import esphome.codegen as cg
+from esphome.components import binary_sensor
+import esphome.config_validation as cv
+from esphome.const import DEVICE_CLASS_OCCUPANCY
+
+from . import CONF_MR60BHA2_ID, MR60BHA2Component
+
+DEPENDENCIES = ["seeed_mr60bha2"]
+
+CONF_PEOPLE_EXIST = "people_exist"
+
+CONFIG_SCHEMA = {
+    cv.GenerateID(CONF_MR60BHA2_ID): cv.use_id(MR60BHA2Component),
+    cv.Optional(CONF_PEOPLE_EXIST): binary_sensor.binary_sensor_schema(
```

Comment:
"has_target" is used by other mmwave sensors (including `mr24hpc1`) and might be a better key for this binary sensor.
```suggestion
    cv.Optional(CONF_HAS_TARGET): binary_sensor.binary_sensor_schema(
```